recipes/default.rb

```diff
@@ -41,20 +41,24 @@ def listen_addr_for(interface, type)
   node.normal['openssh']['server']['listen_address'] = listen_addresses
 end
 
-template 'sshd_ca_keys_file' do
-  source 'ca_keys.erb'
-  path node['openssh']['server']['trusted_user_c_a_keys']
-  mode node['openssh']['config_mode']
-  owner 'root'
-  group node['root_group']
+if node['openssh']['server']['trusted_user_c_a_keys']
```

Comment:
If you add this commit to a PR on its own this could make sense because if these attributes don't exist then one wouldn't want to create these templates. It might make more sense to have these as a guard on the resources though rather than using ruby if statements.

Reply:
I ran into this same issue in my environment. My first thought was also to use a guard. Unfortunately the recipe won't compile if the attributes in the resource don't exist. An if statement around the resource tells chef to not process any part of that resource.